guide/guide/.js/src/main/scala/io/udash/web/guide/views/frontend/demos/FileInputDemo.scala

```diff
@@ -19,7 +23,27 @@ object FileInputDemo extends AutoDemo with CssView {
       FileInput(selectedFiles, acceptMultipleFiles)("files"),
       h4("Selected files"),
       ul(repeat(selectedFiles)(file => {
-        li(file.get.name).render
+        val content = Property(Array.empty[Byte])
+
+        window.setTimeout(() =>
```

Comment:
Why is this necessary? It seems to work without it. Let's keep the examples as simple as possible.

Reply:
Honestly? I haven't remember.

Removed.